bundle/manifests/ibm-websphere-liberty.clusterserviceversion.yaml

```diff
@@ -547,6 +547,11 @@ spec:
         path: conditions
         x-descriptors:
         - urn:alm:descriptor:io.kubernetes.conditions
+      - description: Exposed URI of the application endpoint
```

Comment:
Set a higher order than `.status.binding`, so that the URI is shown first - above the binding secret.